uuid.go

```diff
@@ -362,7 +362,7 @@ type UUIDs []UUID
 
 // Strings returns a string slice containing the string form of each UUID in uuids.
 func (uuids UUIDs) Strings() []string {
-	var uuidStrs = make([]string, len(uuids))
+	uuidStrs := make([]string, len(uuids))
```

Comment:
This is unrelated to the PR.

Reply:
Solved in [54f9572](https://github.com/google/uuid/pull/166/commits/54f95728c20a700436b4f3a8e699a99235748175)